client/.eslintrc.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
+	env: {
+		browser: true,
+		es2021: true
+	},
+	extends: ["metarhia"],
```

Comment:
sonar is missing

Reply:
https://let-is-deploy-lab-2.herokuapp.com/